charts/kyverno/templates/reports-controller/clusterrole.yaml

```diff
@@ -26,13 +26,26 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
+      - ''
+    resources:
+      - secrets
+      - configmaps
+      - namespaces
+    verbs:
+      - get
+      - list
+      - watch
```

Comment:
Why are these permissions required in the cluster role?

Reply:
@eddycharly - correct me if I'm wrong, but these are not required by the report controller. Allowing these by default simplifies the use of reporting, it makes sense to set defaults for configmaps and namespaces, but not secrets. See this issue to remove default secrets permissions https://github.com/kyverno/kyverno/issues/5690.